core/integration/module_test.go

```diff
@@ -4681,40 +4565,312 @@ func TestModuleNamespacing(t *testing.T) {
 	require.JSONEq(t, `{"test":{"fn":["*main.Sub1Obj made 1:yo", "*main.Sub2Obj made 2:yo"]}}`, out)
 }
 
-func TestModuleRoots(t *testing.T) {
-	t.Parallel()
+<<<<<<< HEAD
```

Comment:
Legit failure. I know that it would have been very difficult to spot considering the amount of change that this went through. All good!

Reply:
Yeah this PR has had a tumultuous history at this point 😄. Can finally bring it to a close tomorrow post-release. Fixed all the rebasing now either way.